packages/xod-client/src/user/containers/AccountPane.jsx

```diff
@@ -0,0 +1,91 @@
+import R from 'ramda';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { foldMaybe } from 'xod-func-tools';
+
+import Button from '../../core/components/Button';
+
+import * as Actions from '../actions';
+import * as Selectors from '../selectors';
+
+import AuthForm from '../components/AuthForm';
+
+const AccountPane = ({
+  user,
+  isVisible,
+  compilationsLeft,
+  actions,
+  isAuthorising,
+  authError,
+}) => {
+  if (!isVisible) return null;
+
+  return (
+    <div className="AccountPane">
+      <a
```

Comment:
Looks like a recurring pattern, it would be great to make it a separate component like you did with `Button` :)